### PR TITLE
[cd] Fix release test pipeline

### PR DESCRIPTION
**Pull request by can-anyscale**
I miss one place where the test collection file needs to be convert into a list.
- Pass down the list of test collection files to run_release_test.py

Test:
- CI
- Release test run: https://buildkite.com/ray-project/release-tests-pr/builds/57576

### CODE REVIEW

File: release/ray_release/tests/test_config.py [hunk 1]
```diff
@@ -14,8 +13,8 @@
 from ray_release.exception import ReleaseTestConfigError
 
 _TEST_COLLECTION_FILES = [
-    bazel_runfile("release/release_tests.yaml"),
-    bazel_runfile("release/ray_release/tests/test_collection_data.yaml"),
+    "release_tests.yaml",
+    "ray_release/tests/test_collection_data.yaml",
```
Comment (aslonnie) [inline release/ray_release/tests/test_config.py:17]: why are you removing the `bazel_runfile`? could you keep it to be using bazel runfiles consistently?

File: release/ray_release/config.py [hunk 2]
```diff
@@ -40,12 +40,13 @@
...
 def read_and_validate_release_test_collection(
-    config_files: List[str], schema_file: Optional[str] = None
+    config_files: List[str],
+    schema_file: Optional[str] = None,
 ) -> List[Test]:
     """Read and validate test collection from config file"""
     tests = []
     for config_file in config_files:
-        with open(config_file, "rt") as fp:
+        with open(bazel_runfile(config_file), "rt") as fp:
```
Comment (can-anyscale) [inline release/ray_release/config.py:49]: [1]

File: release/ray_release/tests/test_config.py [hunk 3]
```diff
@@ -14,8 +13,8 @@
 from ray_release.exception import ReleaseTestConfigError
 
 _TEST_COLLECTION_FILES = [
-    bazel_runfile("release/release_tests.yaml"),
-    bazel_runfile("release/ray_release/tests/test_collection_data.yaml"),
+    "release/release_tests.yaml",
+    "release/ray_release/tests/test_collection_data.yaml",
```
Comment (can-anyscale) [inline release/ray_release/tests/test_config.py:17]: bazel_runfile is move to [1]

**Review by aslonnie — APPROVED** [review]

File: release/ray_release/config.py (hunk 2 above)
Comment (aslonnie) [reply release/ray_release/config.py:49]: not sure if this will work though..

Comment (can-anyscale) [reply release/ray_release/config.py:49]: discuss offline, bazel_runtile returns an absolute that works both in bazel runtime and python runtime environment

**Pull request merged**